docs/docs/portals.md

```diff
@@ -63,12 +63,35 @@ This includes event bubbling. An event fired from inside a portal will propagate
 </html>
 ```
 
-A Parent component in `#app-root` would be able to catch an uncaught, bubbling event from the sibling node #modal-root.
+A `Parent` component in `#app-root` would be able to catch an uncaught, bubbling event from the sibling node `#modal-root`.
 
-```js
+```js{20-23,34-41,45,53-55,62-63,66}
+// These two containers are siblings in the DOM
 const appRoot = document.getElementById('app-root');
 const modalRoot = document.getElementById('modal-root');
 
+class Modal extends React.Component {
```

Comment:
Might be worth adding an inline comment about accessibility, in case people copy + paste without reading the other note.

Reply:
What would be the best place to add it? I'll leave that to you for now, running home!